### ISSUE TITLE
X1 Carbon 7 acpidump output

### ISSUE THREAD

**Issue by mrhpearson** [mjg59/thermal_daemon#6]
acpi.out as requested.
[acpi.zip](https://github.com/mjg59/thermal_daemon/files/4560033/acpi.zip)

**Comment by mjg59** [comment]
Are you able to do:

grep -r . /sys/bus/platform/devices/INT3400:00/

before and after doing something that you think should switch between profiles? The tables actually contain two separate DPTF tables which are chosen based on a conditional, I need to dig into what controls that conditional.

**Comment by mrhpearson** [comment]
No problem here you go (going from desk mode to lap mode) - nothing jumped out at me so let me know if you need anything else
Mark
--------
[root@localhost banther]# grep -r . /sys/bus/platform/devices/INT3400:00/
/sys/bus/platform/devices/INT3400:00/odvp10:0
/sys/bus/platform/devices/INT3400:00/uevent:DRIVER=int3400 thermal
/sys/bus/platform/devices/INT3400:00/uevent:MODALIAS=acpi:INT3400:
/sys/bus/platform/devices/INT3400:00/odvp19:0
/sys/bus/platform/devices/INT3400:00/odvp3:0
/sys/bus/platform/devices/INT3400:00/odvp17:0
/sys/bus/platform/devices/INT3400:00/odvp1:0
/sys/bus/platform/devices/INT3400:00/uuids/available_uuids:3A95C389-E4B8-4629-A526-C52C88626BAE
/sys/bus/platform/devices/INT3400:00/uuids/available_uuids:63BE270F-1C11-48FD-A6F7-3AF253FF3E2D
/sys/bus/platform/devices/INT3400:00/uuids/available_uuids:9E04115A-AE87-4D1C-9500-0F3E340BFE75
/sys/bus/platform/devices/INT3400:00/uuids/current_uuid:INVALID
Binary file /sys/bus/platform/devices/INT3400:00/data_vault matches
/sys/bus/platform/devices/INT3400:00/odvp15:0
/sys/bus/platform/devices/INT3400:00/power/runtime_active_time:0
/sys/bus/platform/devices/INT3400:00/power/runtime_status:unsupported
grep: /sys/bus/platform/devices/INT3400:00/power/autosuspend_delay_ms: Input/output error
/sys/bus/platform/devices/INT3400:00/power/runtime_suspended_time:0
/sys/bus/platform/devices/INT3400:00/power/control:auto
/sys/bus/platform/devices/INT3400:00/odvp8:0
/sys/bus/platform/devices/INT3400:00/odvp13:0
/sys/bus/platform/devices/INT3400:00/odvp6:0
/sys/bus/platform/devices/INT3400:00/odvp11:0
/sys/bus/platform/devices/INT3400:00/odvp4:0
/sys/bus/platform/devices/INT3400:00/odvp18:0
/sys/bus/platform/devices/INT3400:00/odvp2:0
/sys/bus/platform/devices/INT3400:00/driver_override:(null)
/sys/bus/platform/devices/INT3400:00/odvp16:0
/sys/bus/platform/devices/INT3400:00/odvp0:0
/sys/bus/platform/devices/INT3400:00/odvp9:0
/sys/bus/platform/devices/INT3400:00/odvp14:0
/sys/bus/platform/devices/INT3400:00/odvp7:0
/sys/bus/platform/devices/INT3400:00/odvp12:0
/sys/bus/platform/devices/INT3400:00/odvp20:0
/sys/bus/platform/devices/INT3400:00/modalias:acpi:INT3400:
/sys/bus/platform/devices/INT3400:00/odvp5:0
[root@localhost banther]# #lapmode
[root@localhost banther]# grep -r . /sys/bus/platform/devices/INT3400:00/
/sys/bus/platform/devices/INT3400:00/odvp10:0
/sys/bus/platform/devices/INT3400:00/uevent:DRIVER=int3400 thermal
/sys/bus/platform/devices/INT3400:00/uevent:MODALIAS=acpi:INT3400:
/sys/bus/platform/devices/INT3400:00/odvp19:0
/sys/bus/platform/devices/INT3400:00/odvp3:0
/sys/bus/platform/devices/INT3400:00/odvp17:0
/sys/bus/platform/devices/INT3400:00/odvp1:0
/sys/bus/platform/devices/INT3400:00/uuids/available_uuids:3A95C389-E4B8-4629-A526-C52C88626BAE
/sys/bus/platform/devices/INT3400:00/uuids/available_uuids:63BE270F-1C11-48FD-A6F7-3AF253FF3E2D
/sys/bus/platform/devices/INT3400:00/uuids/available_uuids:9E04115A-AE87-4D1C-9500-0F3E340BFE75
/sys/bus/platform/devices/INT3400:00/uuids/current_uuid:INVALID
Binary file /sys/bus/platform/devices/INT3400:00/data_vault matches
/sys/bus/platform/devices/INT3400:00/odvp15:0
/sys/bus/platform/devices/INT3400:00/power/runtime_active_time:0
/sys/bus/platform/devices/INT3400:00/power/runtime_status:unsupported
grep: /sys/bus/platform/devices/INT3400:00/power/autosuspend_delay_ms: Input/output error
/sys/bus/platform/devices/INT3400:00/power/runtime_suspended_time:0
/sys/bus/platform/devices/INT3400:00/power/control:auto
/sys/bus/platform/devices/INT3400:00/odvp8:0
/sys/bus/platform/devices/INT3400:00/odvp13:0
/sys/bus/platform/devices/INT3400:00/odvp6:0
/sys/bus/platform/devices/INT3400:00/odvp11:0
/sys/bus/platform/devices/INT3400:00/odvp4:0
/sys/bus/platform/devices/INT3400:00/odvp18:0
/sys/bus/platform/devices/INT3400:00/odvp2:0
/sys/bus/platform/devices/INT3400:00/driver_override:(null)
/sys/bus/platform/devices/INT3400:00/odvp16:0
/sys/bus/platform/devices/INT3400:00/odvp0:0
/sys/bus/platform/devices/INT3400:00/odvp9:0
/sys/bus/platform/devices/INT3400:00/odvp14:0
/sys/bus/platform/devices/INT3400:00/odvp7:0
/sys/bus/platform/devices/INT3400:00/odvp12:0
/sys/bus/platform/devices/INT3400:00/odvp20:0
/sys/bus/platform/devices/INT3400:00/modalias:acpi:INT3400:
/sys/bus/platform/devices/INT3400:00/odvp5:0

**Comment by mjg59** [comment]
Yeah, none of the variables changed there - the firmware isn't telling us that there's a difference between the states. Hmm.
